src/icastats_shared.c

```diff
@@ -27,6 +27,7 @@
 #include "icastats.h"
 #include "init.h"
 
+#define NOT_INITIALIZED (-1)
```

Comment:
I would not define a macro for this. The `-1` is just a normal return value of a syscall.

Reply:
Yes, I added the line because this define was used in the old code. But I agree, as we are using the define only here, we may indeed just check for -1.

Reply:
Fixed.